package.json

```diff
@@ -33,51 +34,20 @@
     "type": "git",
     "url": "https://github.com/devtools-html/perf.html"
   },
-  "devDependencies": {
+  "dependencies": {
     "array-move": "^1.0.0",
     "array-range": "^1.0.1",
-    "babel-cli": "^6.24.0",
-    "babel-core": "^6.24.0",
-    "babel-eslint": "^7.2.1",
-    "babel-loader": "^6.4.1",
     "babel-plugin-transform-runtime": "^6.23.0",
```

Comment:
No problem.

FWIW I did try to build with webpack in an isolated environment.  And, webpack didn't complain, which I thought it should have if something was wrong... but I seem to have been mistaken about that.  I assume the hackish way I did the test had something to do with this.